lib/rules/declaration-property-max-values/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const matchesStringOrRegExp = require('../../utils/matchesStringOrRegExp');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const valueParser = require('postcss-value-parser');
```

Comment:
```suggestion
const valueParser = require('postcss-value-parser');

const matchesStringOrRegExp = require('../../utils/matchesStringOrRegExp');
const report = require('../../utils/report');
const ruleMessages = require('../../utils/ruleMessages');
```

Let's move the external `require` to the top (and separate it out). This seems to be the convention we're adopting.

(We should probably use Prettier or ESLint to do this for us).